docs/package.json

```diff
@@ -29,6 +30,7 @@
     "mdx-prism": "^0.3.3",
     "next": "12",
     "next-plugin-preval": "^1.2.1",
+    "next-sitemap": "^1.6.203",
```

Comment:
Do we need to declare this dependency is we are using `npx` in the script? Would `npx` always pull the latest version and not use the version defined here?

Reply:
+1

Reply:
My understanding of `npx` was that it would first look locally to see if the package exists in `node_modules` and then only download from npm if it's not present.